test/evergreen.yml

```diff
@@ -5291,7 +5292,7 @@ tasks:
       - func: "fetch artifacts"
         vars:
           dependent_task: perf-test-long-500m-btree-populate
-          destination: "wiredtiger/cmake_build/bench/wtperf/WT_TEST_0_0"
```

Comment:
This is the section I'm not 100% happy with, as we've lost the clarity that `WT_TEST_0_0` is the folder being fetched. Suggestions here are welcome